query-engine/core/src/executor/loader.rs

```diff
@@ -37,82 +32,22 @@ pub async fn load(
     }
 }
 
-pub fn db_name(source: &Datasource, url: &str) -> crate::Result<String> {
```

Comment:
Do we still have an easy way to parse the db name out of a datasource? I need that atm for this PR https://github.com/prisma/prisma-engines/pull/3581

Reply:
Schema name is not a thing on every connector (sqlite, mongo, etc.), so my bias would be to do it case by case where we need it, in a connector-specific way and not in core code.